recipes/geographiclib/all/conanfile.py

```diff
@@ -23,9 +24,34 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    @property
+    def _min_compiler_version_default_cxx11(self):
+        # Minimum compiler version having c++ standard >= 11
+        return {
+            "apple-clang": "3.3",
+            "gcc": "4.9",
+            "clang": "6",
+            "Visual Studio": "14",  # guess
+        }.get(str(self.settings.compiler), False)
+
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        if tools.Version(self.version) >= "1.51":
+            if self.settings.compiler.cppstd:
+                tools.min_cppstd(self, 11)
+
+            def lazy_lt_semver(v1, v2):
```

Comment:
Another variant https://github.com/conan-io/conan/issues/8002 (just to keep track of all of them)